src/cli/run_command.zig

```diff
@@ -225,26 +226,70 @@ pub const RunCommand = struct {
 
     const log = Output.scoped(.RUN, false);
 
-    pub fn runPackageScript(
+    inline fn spawn(
+        allocator: std.mem.Allocator,
+        name: string,
+        cwd: string,
+        env: *DotEnv.Loader,
+        argv: [*:null]?[*:0]const u8,
+    ) !?std.os.pid_t {
+        var flags: i32 = bun.C.POSIX_SPAWN_SETSIGDEF | bun.C.POSIX_SPAWN_SETSIGMASK;
+        if (comptime Environment.isMac) {
+            flags |= bun.C.POSIX_SPAWN_CLOEXEC_DEFAULT;
+        }
+
+        var attr = try PosixSpawn.Attr.init();
+        defer attr.deinit();
+        try attr.set(@intCast(flags));
+        try attr.resetSignals();
+
+        var actions = try PosixSpawn.Actions.init();
+        defer actions.deinit();
+        try actions.inherit(0);
```

Comment:
I don't think we can ship this with `inherit`. It's going to be too noisy.

Reply:
sounds reasonable &minus; will work on this now